browser/images/dark/lc_alignverticalcenter.svg

```diff
@@ -1 +0,0 @@
-<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="m7.99982 5.515625c-.0115-.454318.540283-.686847.857422-.361328l2.144531 2.148437-.0018-5.777343c-.017923-.350177.241538-.525391.500027-.525391s.51795.175214.5.525391l-.0022 5.777343 2.144531-2.148437c.317139-.325519.868926-.09299.857422.361328-.0041.130048-.05875.25337-.152344.34375l-2.912092 2.916016c-.160599.173289-.266379.223009-.435497.224609-.169118-.0016-.274898-.05132-.435547-.224609l-2.912109-2.916016c-.09359-.09038-.148244-.213702-.152344-.34375z" fill="#1e8bcd" fill-rule="evenodd"/><g fill="#fafafa" transform="scale(-1 1)"><rect height="1" ry=".5" width="18" x="-21" y="10"/><rect height="1" ry=".5" width="11" x="-14" y="13"/></g><path d="m7.99982 18.484375c-.0115.454318.540283.686847.857422.361328l2.144531-2.148437-.0018 5.777343c-.017923.350177.241538.525391.500027.525391s.51795-.175214.5-.525391l-.0022-5.777343 2.144531 2.148437c.317139.325519.868926.09299.857422-.361328-.0041-.130048-.05875-.25337-.152344-.34375l-2.912092-2.916016c-.160599-.173289-.266379-.223009-.435497-.224609-.169118.0016-.274898.05132-.435547.224609l-2.912109 2.916016c-.09359.09038-.148244.213702-.152344.34375z" fill="#1e8bcd" fill-rule="evenodd"/></svg>
```

Comment:
This SVG file doesn't appear to be mentioned in the aliases anywhere. Was it meant to be deleted?